src/meltano/core/project_init_service.py

```diff
@@ -91,7 +94,9 @@ def create_files(self, add_discovery=False):
             add_discovery: Add discovery.yml file to created project
         """
         click.secho("Creating project files...", fg="blue")
-        click.echo(f"  {self.project_name}/")
+
```

Comment:
GitHub won't let me put a suggestion above this line, but you'll need to:
- Update the signature of `create_files` to take the `project: Project` parameter, and use it instead of `self.project`.
- Update the signature of `create_dot_meltano_dir` to take the `project: Project` parameter, and use it instead of `self.project`.
- Update the signature of `create_system_database` to take the `project: Project` parameter, and use it instead of `self.project`.
- Update `Project.activate(self.project)` to `Project.activate(project)`.
- Update `return self.project` to `return project`.
- Update `for path in plugin.create_files(self.project):` to `for path in plugin.create_files(project):`.